package.json

```diff
@@ -50,6 +50,7 @@
     "input-plugin-quote": "0.0.1",
     "input-plugin-text": "^0.0.5",
     "input-plugin-textarea": "^0.0.1",
+    "node-dir": "^0.1.12",
```

Comment:
This is in the direct deps stack because it is used by [load.js](https://github.com/punchcard-cms/content-types/blob/master/lib/content-types/load.js) which is not a development file, it is part of the main system